maestro-cli/src/main/java/maestro/cli/device/DeviceService.kt

```diff
@@ -90,13 +90,15 @@ object DeviceService {
             .build()
 
         IdbIOSDevice(channel).use { iosDevice ->
-            MaestroTimer.retryUntilTrue(timeoutMs = 2000) {
+            println("Waiting for idb service to start..")
```

Comment:
would be also useful to have date timestamps  in those logs

Reply:
This is not a log, this is a message to the user so that they know why maestro isn't showing a result yet.
I agree we should build some logging facility. But this is not the right place.